src/game/behaviors/bowling_ball.inc.c

```diff
@@ -241,7 +241,8 @@ void bhv_free_bowling_ball_init(void) {
 }
 
 void bhv_free_bowling_ball_roll_loop(void) {
-    s16 collisionFlags = object_step();
+    // See comment below about collisionFlags
+    /* s16 collisionFlags = */ object_step();
```

Comment:
you can just remove it at this point, the inline comment looks strange
```suggestion
    object_step();
```